index.php

```diff
@@ -145,24 +145,28 @@
                                 <td>$18</td>
                             </tr>
                             <tr>
-                                <td>HOT OIL MANICURE</td>
-                                <td>$25</td>
+                                <td>SOFTGEL MANICURE</td>
+                                <td>$46</td>
                             </tr>
                             <tr>
                                 <td>GEL POLISH CHANGE</td>
                                 <td>$32</td>
                             </tr>
                             <tr>
-                                <td>REGULAR GEL MANICURE</td>
+                                <td>GEL MANICURE</td>
                                 <td>$36</td>
                             </tr>
                             <tr>
-                                <td>ORGANIC SPA MANICURE</td>
-                                <td>$30</td>
+                                <td>ORGANIC SPA MANICURE (REG)</td>
+                                <td>$28</td>
+                            </tr>
+                            <tr>
+                                <td>ORGANIC SPA MANICURE (GEL)</td>
+                                <td>$46</td>
                             </tr>
                             <tr>
                                 <td>EXTENDED WEAR POLISH</td>
-                                <td>+$3</td>
```

Comment:
Items like this with `+` in front of the price should be displayed after all items without `+`.

Reply:
fixed here: https://github.com/jisanjung/Glitter-and-Polish/pull/1/commits/9b6759c9bad4db07ffc748aec7143ed63387ea18